docs/pages/installation/minikube.md

```diff
@@ -108,6 +108,15 @@ To establish connectivity between a particular Meshery Adapter and Kubernetes se
  mesheryctl system config minikube -t ~/Downloads/auth.json
  </div></div>
  </pre>
+<br/>
```

Comment:
This optional, but helpful step should be clearly marked as such and presented in such a way that the lazy reader doesn't assume it is a mandatory step.